packages/utils/src/random.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Maybe you want to mention whether the value is excluding / including the passed arguments?